apps/devhub/views.py

```diff
@@ -1196,6 +1197,19 @@ def version_add(request, addon_id, addon):
                 form.cleaned_data['nomination_type']):
             addon.update(status=form.cleaned_data['nomination_type'])
         url = reverse('devhub.versions.edit', args=[addon.slug, str(v.id)])
+
+        if not addon.is_listed:  # Not listed? Do we need to sign?
+            # We need to get to the file to check its validation.
+            # We're submitting a new version here, so it has only one file.
+            file_ = v.files.get()
```

Comment:
Can't versions have multiple files for different platforms? Will that not be the case here? (I see your comment) Do we need to sign other files for the same version elsewhere? What's the code flow for adding files for other platforms?